### PR TITLE
scan: support multisend msg

### LINKED ISSUE #1306: Support Showing MultiSend Msg (UI + Wireup)
- Send this msg to the chain.
- Design & Implement UI msg on TxHomePage
- Design & Implement UI msg on TxIndexPage & LatestTx
- Add this msg to TxSub & Wire up

Note: If you struggle with design, feel free to ask @prin-r or @evilpeach or @sorawit.

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1306 

image

image

### CODE REVIEW

File: scan/src/components/TxIndexPageTable.re [hunk 1]
```diff
@@ -855,6 +855,89 @@ let renderSetWithdrawAddress = (set: TxSub.Msg.SetWithdrawAddress.t) => {
...
 
+let renderMultiSend = (tx: TxSub.Msg.MultiSend.t) => {
+  <Col size=Styles.thirdCol alignSelf=Col.Start>
+    <VSpacing size=Spacing.sm />
+    <div className=Styles.topicContainer>
+      <Text
+        value="NUMBER OF MESSAGES"
+        size=Text.Sm
+        weight=Text.Thin
+        spacing={Text.Em([@reason.preserve_braces] 0.06)}
```
Comment (evilpeach) [inline scan/src/components/TxIndexPageTable.re]: Why use `@reason.preserve_braces`?

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Review by prin-r — CHANGES_REQUESTED** [review]
-

File: scan/src/components/TxIndexPageTable.re [hunk 2]
```diff
@@ -901,6 +901,84 @@ let renderVote = (vote: TxSub.Msg.Vote.t) => {
...
+       )
+     ->Belt_List.toArray
+     ->React.array}
+    <div className=Styles.topicContainer>
+      <Text value="Outputs" size=Text.Sm weight=Text.Semibold spacing={Text.Em(0.06)} />
+    </div>
+    <VSpacing size=Spacing.lg />
+    {tx.outputs
+     ->Belt_List.map(output =>
+         <div>
```
Comment (evilpeach) [inline scan/src/components/TxIndexPageTable.re]: add key

File: scan/src/components/TxIndexPageTable.re [hunk 3]
```diff
@@ -901,6 +901,84 @@ let renderVote = (vote: TxSub.Msg.Vote.t) => {
...
+    </div>
+    <VSpacing size=Spacing.lg />
+    <VSpacing size=Spacing.lg />
+    <div className=Styles.topicContainer>
+      <Text value="Inputs" size=Text.Sm weight=Text.Semibold spacing={Text.Em(0.06)} />
+    </div>
+    <VSpacing size=Spacing.lg />
+    {tx.inputs
+     ->Belt_List.map(input =>
+         <div>
```
Comment (evilpeach) [inline scan/src/components/TxIndexPageTable.re]: add key

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: scan/src/components/TxIndexPageTable.re [hunk 4]
```diff
@@ -901,6 +901,84 @@ let renderVote = (vote: TxSub.Msg.Vote.t) => {
   </Col>;
 };
 
+let renderMultiSend = (tx: TxSub.Msg.MultiSend.t) => {
+  <Col size=Styles.thirdCol alignSelf=Col.Start>
+    <VSpacing size=Spacing.sm />
+    <div className=Styles.topicContainer>
+      <Text value="NUMBER OF MESSAGES" size=Text.Sm weight=Text.Thin spacing={Text.Em(0.06)} />
```
Comment (sorawit) [inline scan/src/components/TxIndexPageTable.re]: Remove this thing

File: scan/src/components/msgs/Msg.re [hunk 5]
```diff
@@ -1002,6 +1002,32 @@ let make = (~msg: TxSub.Msg.t, ~width: int) => {
         <Text value="BAND" weight=Text.Regular code=true nowrap=true block=true />
       </div>
     </div>
+  | MultiSend({inputs, outputs}) =>
+    let firstTx = inputs |> Belt_List.get(_, 0) |> Belt_Option.getExn;
```
Comment (sorawit) [inline scan/src/components/msgs/Msg.re]: Use BeltList's getExn

File: scan/src/subscriptions/TxSub.re [hunk 6]
```diff
@@ -762,6 +786,9 @@ module Msg = {
     | Deposit(deposit) => deposit.depositor
     | Vote(vote) => vote.voterAddress
     | WithdrawCommission(withdrawal) => withdrawal.validatorAddress
+    | MultiSend(tx) =>
+      let firstTx = tx.inputs |> Belt_List.get(_, 0) |> Belt_Option.getExn;
```
Comment (sorawit) [inline scan/src/subscriptions/TxSub.re]: Use BeltList's getExn

**Review by sorawit — DISMISSED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**